interrupt/maven/src/main/java/iotdk/example/Interrupt.java

```diff
@@ -95,6 +95,8 @@ public static void initPlatform(){
             pinNumber = 26;
         if(platform.equals(Platform.INTEL_JOULE_EXPANSION))
             pinNumber = 26;
+        if (platform.equals(Platform.IEI_TANK))_
```

Comment:
What is the use of underscore here? Was this intentional? 
Everything else LGTM.

Reply:
> What is the use of underscore here? Was this intentional?
> Everything else LGTM.

No, I pressed it unintentionally. Just fixed it. Thanks, Sasmita!